ide/coqide/coqide_ui.ml

```diff
@@ -98,9 +99,11 @@ let init () =
 \n    <menuitem action='Forward' />\
 \n    <menuitem action='Backward' />\
 \n    <menuitem action='Run to cursor' />\
+\n    <menuitem action='Reset' />\
 \n    <menuitem action='Run to end' />\
+\n    <menuitem action='Fully check' />\
 \n    <menuitem action='Interrupt' />\
-\n    <menuitem action='Reset' />\
+\n    <separator/>\
```

Comment:
Why did you drop Reset?

Reply:
I didn’t drop it, I just put it before, in a place which is more logical to me.